### PR TITLE
Add keys migrate docs

### PR DESCRIPTION
**Pull request by dreamer-zq**

### CODE REVIEW

**Review by SegueII — APPROVED** [review]

File: migrate/keys.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+2. Use irishub 1.0 to import keystore file
+
+    ```bash
+    iris keys import n2 test1_key --keyring-backend file 
+    ```
+
+3. Verify the imported key information
+
+    ```bash
+    iris keys show n2 --keyring-backend file
```
Comment (chengwenxi) [inline migrate/keys.md:49]: no need to specify `keyring-backend`

**Review by chengwenxi — COMMENTED** [review]
Could you replace `1.0` with `v1.0`?

File: migrate/keys.md (hunk 1 above)
Comment (dreamer-zq) [reply migrate/keys.md:49]: This is just an example

**Comment by seekelvis** [comment]
migrate/keys.md: 3 : The keyfile(private key)of

**Comment by seekelvis** [comment]
Missing spaces

**Review by chengwenxi — APPROVED** [review]

**Pull request merged**